packages/react-sdk/README.md

```diff
@@ -136,15 +147,32 @@ export const CreateClient: React.FC<{ signer: Signer }> = ({ signer }) => {
 };
 ```
 
-#### Configure the client
+### Configure the client
 
-To learn more about client configuration options, see [Configure the client](https://github.com/xmtp/xmtp-js#configure-the-client) in the `xmtp-js` SDK docs.
+The client's network connection and key storage method can be configured with these optional parameters of `Client.create`:
 
-#### Create a client with private keys
+| Parameter                 | Default                                                                         | Description                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ------------------------- | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| appVersion                | undefined                                                                       | Add a client app version identifier that's included with API requests.<br/>For example, you can use the following format: appVersion: APP_NAME + '/' + APP_VERSION.<br/>Setting this value provides telemetry that shows which apps are using the XMTP client SDK. This information can help XMTP developers provide app support, especially around communicating important SDK updates, including deprecations and required upgrades. |
+| env                       | dev                                                                             | Connect to the specified XMTP network environment. Valid values include dev, production, or local. For important details about working with these environments, see [XMTP production and dev network environments](https://github.com/xmtp/xmtp-web/blob/main/packages/react-sdk/README.md#xmtp-production-and-dev-network-environments).                                                                                              |
```

Comment:
I used to be able to anchor link within a readme ... but now VS Code doesn't allow it. I wonder why?